packages/zoe/src/contractSupport/atomicTransfer.js

```diff
@@ -52,6 +43,8 @@ export const TransferPartShape = M.splitArray(
  * which will remain helpers. These helper are for convenience
  * in expressing atomic rearrangements clearly.
  *
+ * @deprecated use the zcf builtin instead
+ *
  * @param {ZCF} zcf
  * @param {TransferPart[]} transfers
  */
```

Comment:
If the new code has the same semantics, why not redefine the old code to delegate to the new code?

```suggestion
 */
 export const atomicRearrange = (zcf, transfers) => zcf.atomicRearrange(transfers);
```

Reply:
Doh!  of course.